content/guides/tooling/cli/quick-start/0-index.md

```diff
@@ -26,7 +26,8 @@ This guide will take you through the following steps.
 2. [Install and configure][two] the CLI
 3. [Run commands][three] with the CLI
 4. Use [options and bulk commands][four]
-5. [Next steps][five]
+5. Explore using [Powershell scripts with the CLI][five]
```

Comment:
I miss the info about what should be explored - for example `Explore automation using Powershell scripts`